EXTERNAL_IMAGES.md

```diff
@@ -40,6 +40,16 @@ export function decorateMain(main) {
 ```
 
 ## How does this work?
-During the page rendering process, the frontend code replaces the anchor tags identified as exteernal images on the page with the `picture` tags with `src`/`srcset` attributes set as the external image's url as specified in the external image link placed on the Word / Google Document during the page authoring process.
+During the page rendering process, the frontend code replaces the anchor tags identified as external images on the page with the `picture` tags with `src`/`srcset` attributes set as the external image's url as specified in the external image link placed on the Word / Google Document during the page authoring process.
 
 Authors can optionally specify query paramaters in the hyperlinked external url and they would be retained in the `picture` tag's `src`/`srcset` attributes. These are useful for specifying image delivery parameters such as image width, height, format, etc. as understood by the external image delivery service.
+
+## Extended Usecase of leveraging DM OpenAPI for Streamlining AEM Franklin Development
+
+For clients utilizing the `AEM Assets` repository to distribute their assets, the burden of maintaining asset integrity due to frequent edits, updates, and deletions performed within AEM is alleviated. Thanks to `Dynamic Media OpenAPI`, synchronization with every DOCX document becomes seamless.
+
+Simply incorporate the necessary `asset-library` configuration into AEM sidekick plugin, as outlined in [here](https://github.com/anuraggupta228/franklinvideo/commit/04d710632f5c614ba7efd52cd9f720f6f4cb58dd)
+
+Once `asset-library` in integrated then simply perform copy/paste through plugin for any asset into target DOCX. Here is one crafted sample on how it might look like - [Check Here](https://docs.google.com/document/d/1fSoGW0X27sZD6KTaZ--7KwNRPBHwO5qjuAnQS1jDa9U/edit?usp=sharing)
```

Comment:
Same as prior suggestion, let's also have the example doc in this rep's Sharepoint. I think you'd already have access. You can add an example page to demonstrate.

Reply:
added the doc ref as [here](https://adobe.sharepoint.com/:w:/s/HelixProjects/EcC9uxWYlVRCsKzqbjnV44UByeCoBwcRsjz0iPpZgf6zjw?e=Xet9U6) but I cannot make that doc public so have just limited the access to adobe only. 

If you can wish to make it public, can you please do it and share the public ref with me, i'll do the update.

Reply:
Please place the example word doc in "aem-assets" folder as that's the mountpoint of this repo. No need to add doc link. Instead place the pgae link. Its possible to view the doc source from the page link.

Reply:
done